qrisk/tests/test_stats.py

```diff
@@ -2,75 +2,154 @@
 
```

Comment:
since this is a small repo and porting wouldn't be hard, I am wondering why we chose to use nose instead of pytest. nose is currently unsupported so I would prefer not to start a new project using it.